StoreSpidLogs/index.ts

```diff
@@ -54,24 +57,37 @@ const contextTransport = new AzureContextTransport(() => logger, {
 });
 winston.add(contextTransport);
 
-const OutputBinding = t.interface({
-  spidRequestResponse: SpidBlobItem
-});
-
-export type OutputBinding = t.TypeOf<typeof OutputBinding>;
+export interface IOutputBinding {
+  spidRequestResponse: SpidBlobItem;
+}
 
 /**
  * Store SPID request / responses, read from a queue, into a blob storage.
  */
 export async function index(
   context: Context,
   spidMsgItem: SpidMsgItem
-): Promise<void | OutputBinding> {
+): Promise<void | IOutputBinding> {
   logger = context.log;
+  const encryptedMsgItem: SpidMsgItem = {
+    createdAt: spidMsgItem.createdAt,
+    createdAtDay: spidMsgItem.createdAtDay,
+    fiscalCode: spidMsgItem.fiscalCode,
+    ip: spidMsgItem.ip,
+    requestPayload: new NodeRSA(rsaPublicKey).encrypt(

```

Comment:
I've checked the implementation a little and it looks like you can create just one instance of NodeRSA (at top level)
to call encrypt multiple times. can you write a test that just does this ? encrypt multiple times then decrypt with another instance